GiTinder/Services/SettingsServices.cs

```diff
@@ -42,58 +44,82 @@ private bool SettingsExists(string username)
             return _context.Settings.Any(s => s.Username == username);
         }
 
-        public virtual void UpdateAndSaveSettingsFoundByUserToken(Settings settings, string usertoken)
+        public virtual void UpdateAndSaveSettingsFoundByUserToken(Settings newSettings, string usertoken)
         {
             var foundSettings = FindSettingsWithLanguagesByUserToken(usertoken);
```

Comment:
@FilipTeply what is the purpose of this line?
if you remove it, what is the purpose of this whole method?

Reply:
the context savechanges can go into updateSettings and then we have only one method